packages/ibm-products-styles/src/components/UserAvatar/_user-avatar.scss

```
@@ -0,0 +1,60 @@
+//
+// Copyright IBM Corp. 2024, 2024
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+// Standard imports.
+@use '../../global/styles/project-settings' as *;
+@use '../../global/styles/mixins';
+@use '@carbon/colors' as *;
+
+// Other Carbon settings if needed
+// TODO: @use '@carbon/styles/scss/grid';
+// or
+// TODO: @use '@carbon/react/scss/grid';
+
+// UserAvatar uses the following Carbon for IBM Products components:
+// TODO: @use(s) of IBM Products component styles used by UserAvatar
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+
+$block-class: #{$pkg-prefix}--user-avatar;
+
+.#{$block-class} {
+  display: flex;
+  width: 4rem;
+  height: 4rem;
+  align-items: center;
+  justify-content: center;
+  border: 0.5px solid transparent;
+  border-radius: 100%;
+  outline: none;
+  outline-offset: 3px;
+}
+
+@mixin setBgColor($buttonColor) {
```

Pretty minor and doesn’t affect whether it works, but think it makes more sense to just call this `$color`?

```suggestion
@mixin setBgColor($color) {
```

---

Done the required changes.